backend/src/main/java/develup/domain/solution/Title.java

```diff
@@ -1,9 +1,46 @@
 package develup.domain.solution;
 
+import java.util.Objects;
+import develup.api.exception.DevelupException;
+import develup.api.exception.ExceptionType;
 import jakarta.persistence.Column;
 import jakarta.persistence.Embeddable;
-import jakarta.validation.constraints.NotBlank;
 
 @Embeddable
-public record Title(@NotBlank @Column(name = "title") String value) {
+public class Title {
+
+    @Column(name = "title")
+    private final String value;
+
+    protected Title() {
+        this.value = "";
+    }
+
+    public Title(String value) {
+        validate(value);
+        this.value = value;
+    }
+
+    private void validate(String value) {
+        if (value.isBlank()) {
+            throw new DevelupException(ExceptionType.INVALID_TITLE);
+        }
+    }
```

Comment:
[Approve]

구체적으로 적으면 좋을 것 같아요

```suggestion
    private void validateIsNotBlank(String value) {
        if (value.isBlank()) {
            throw new DevelupException(ExceptionType.INVALID_TITLE);
        }
    }
```